src/bluesky_stomp/messaging.py

```diff
@@ -224,7 +236,12 @@ def _send_bytes(
             )
         if correlation_id:
             headers = {**headers, CORRELATION_ID_HEADER: correlation_id}
-        self._conn.send(headers=headers, body=message, destination=destination)  # type: ignore
+
+        tracer = get_tracer("_send_bytes")
```

Comment:
it would be better to have the tracer at the class level as it should be a grouping of traces (spans) relating to the same entity covering its functions/methods. THis makes it easier to find/search for related things in the Jaeger GUI